components/eam/src/physics/cam/ndrop.F90

```diff
@@ -505,49 +490,40 @@ subroutine dropmixnuc( &
    wtke(:,:)    = 0._r8
    factnum(:,:,:) = 0._r8
 
+
    ! overall_main_icol_loop
    do icol = 1, ncol
```

Comment:
Ah, I guess we are keeping the loop over columns. This won't be ported to C++, however.